templates/validate-h-entry.html.php

```diff
@@ -35,7 +35,9 @@
 				<div class="alert alert-warning">
 					<p>The parsed <code>name</code> name is longer than the content, which is usually a sign is it malformed due to being implicitly rather than explicitly parsed.</p>
 
-					<p>You should always manually specify what the <code>name</code> of a post is. If it doesn’t have one, make the name the same as the content, e.g. <code>&lt;div class=&quot;e-content p-name&quot;>…</code></p>
+					<p>You should always manually specify what the <code>name</code> of a post is. 
+					   If it doesn’t have one, make the name explicitly empty, 
+					   e.g. <code>&lt;span class=&quot;p-name&quot;>&lt;/span>…</code></p>
```

Comment:
```suggestion
					   e.g. <code>&lt;span class="p-name"&gt;&lt;/span&gt;…</code>
					</p>
```

The tag opening and closing are invalid elements, but the quotes I think are safe, and only get escaped for buggy implementations of the spec AFAIK.